src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  // eslint-disable-next-line prefer-promise-reject-errors
+  setTimeout(() => reject('First promise was rejected'), 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.body.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.body.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
The second promise resolves on both click and contextmenu events. This means that the promise can resolve multiple times, which is not typical behavior for promises. Consider using a flag to ensure it resolves only once.